src/util/logfile_handler.cpp

```diff
@@ -0,0 +1,47 @@
+#include "logfile_handler.hpp"
+
+#include <sstream>
+
+wr::LogfileHandler::LogfileHandler()
```

Comment:
You might want to consider giving `wr::LogfileHandler::LogfileHandler(std::filesystem::path& dir_path, std::string& file_name)` default argument values to ommit the argumentless constructor to avoid issues when updating on of the constructors